.github/workflows/group-35-to-311-ci.yml

```diff
@@ -51,27 +51,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # Create a mtrix to fully test all combinations for version 3.5 (LTS)
         database: ['mariadb', 'pgsql']
-        moodle-branch: ['MOODLE_35_STABLE', 'MOODLE_36_STABLE', 'MOODLE_37_STABLE', 'MOODLE_38_STABLE']
+        moodle-branch: ['MOODLE_35_STABLE']
```

Comment:
Would it make more sense to make sure each lts version is in here, and then just the odds in the include?

Whatever we do here is going to be used as the pattern for the others

Reply:
Possibly not as the matrix might include different versions of node or php across older versions

Reply:
3.5 and 3.9 have different PHP requirements, so they are not going to fit in the same matrix.